sympy/functions/combinatorial/tests/test_comb_numbers.py

```diff
@@ -653,3 +653,12 @@ def test_motzkin():
     assert motzkin.find_motzkin_numbers_in_range(-10, 40) == [1, 1, 2, 4, 9, 21]
     assert motzkin.motzkin(1) == 1
     assert motzkin.motzkin(8) == 127
+    raises(ValueError, lambda: motzkin.motzkin(77.58))
+    raises(ValueError, lambda: motzkin.motzkin(x))
+    raises(ValueError, lambda: motzkin.motzkin(-8))
+    raises(ValueError, lambda: motzkin.find_first_n_motzkin(-7))
+    raises(ValueError, lambda: motzkin.find_first_n_motzkin(x))
+    raises(ValueError, lambda: motzkin.find_first_n_motzkin(112.8))
+    raises(ValueError, lambda: motzkin.is_motzkin(12.1))
```

Comment:
The is_motzkin should just return False for these cases.

Reply:
@oscargus  i have made the changes according to your suggestions please review it.